cpp_api/call_bmc.cpp

```diff
@@ -0,0 +1,40 @@
+// Test file to try loading a GOTO-model into memory and running a sample verification run on it.
```

Comment:
Is there a reason we aren't using `catch` for this test?
Shouldn't tests be in a separate directory from the implementation?

Reply:
There's no reason why we can't have unit tests, but for a first iteration I planned to use this as a binary driver for a regression test, which I though might be easier.

Reply:
It's important to think about testing from the start, both unit and regression tests, and rust integration tests.